components/file_preview_modal/file_preview_modal_image_controls/file_preview_modal_image_controls.tsx

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {ChangeEvent, memo, useEffect, useState} from 'react';
+import {PlusIcon, MinusIcon} from '@mattermost/compass-icons/components';
+import {useIntl} from 'react-intl';
+import {clamp} from 'lodash';
+
+import {minZoom, zoom} from '../image_preview';
+import './file_preview_modal_image_controls.scss';
+
+let zoomInButtonDisabled = false;
+let zoomOutButtonDisabled = true;
+
+export type ZoomValue = 'A' | 'W' | 'H' | number;
+
+interface Props {
+    toolbarZoom: ZoomValue;
+    setToolbarZoom: (toolbarZoom: ZoomValue) => void;
+}
+
+const FilePreviewModalImageControls = ({toolbarZoom, setToolbarZoom}: Props) => {
+    // Initial variables and constants
+    // zoom text
+    const [zoomText, setZoomText] = useState<string>();
+    const [selectedZoomValue, setSelectedZoomValue] = useState<ZoomValue | 'customZoom'>();
+
+    const plusSign = (
+        <PlusIcon
+            size={16}
+            color='currentColor'
+        />
+    );
+
+    const minusSign = (
+        <MinusIcon
+            size={16}
+            color='currentColor'
+        />
+    );
```

Comment:
these can just go into the return value. No need to store them in variables upfront